packages/orbit-components/src/InputField/InputField.stories.tsx

```diff
@@ -10,236 +11,236 @@ import ServiceLogo from "../ServiceLogo";
 import RenderInRtl from "../utils/rtl/RenderInRtl";
 import Tag from "../Tag";
 import { SPACINGS_AFTER } from "../common/consts";
+import type { Name } from "../ServiceLogo/types";
 
 import InputField from ".";
 
 const getIcon = (source: string | null) => source && Icons[source];
 
-export default {
-  title: "InputField",
-};
-
-export const DefaultInput = ({ label, value, placeholder }) => {
-  return (
-    <InputField label={label} value={value} placeholder={placeholder} onChange={action("change")} />
-  );
+type InputFieldPropsAndCustomArgs = React.ComponentProps<typeof InputField> & {
+  serviceLogoName?: Name;
+  grayScale?: boolean;
 };
 
-DefaultInput.story = {
-  name: "Default input",
+const meta: Meta<InputFieldPropsAndCustomArgs> = {
+  title: "InputField",
+  component: InputField,
 
   parameters: {
-    info: "Some description about this type of InputField in general.",
+    controls: {
+      exclude: ["onChange"],
+    },
   },
-};
 
-DefaultInput.args = {
-  label: "Label",
-  value: "",
-  placeholder: "Placeholder",
-};
-
-export const NumberInput = ({ label, value, placeholder, maxValue, minValue }) => {
-  return (
-    <InputField
-      type="number"
-      value={value}
-      label={label}
-      placeholder={placeholder}
-      maxValue={maxValue}
-      minValue={minValue}
-      onChange={action("change")}
-    />
-  );
+  args: {
```

Comment:
What about adding arg `helpclosable` to show that user can have this type of help tooltip as well?